sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+---
+friendly_name: Data from external source Adjust
+description: >
+  This table represents data downloaded from Adjust
+owners:
+  - mhirose@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_adjust_derived
+  date_partition_parameter: submission_date
+  parameters: ["submission_date:DATE:{{ds}}"]
```

Comment:
I think if you specify the `date_partition_parameter` then you don't need `parameters` here.